keycloak/keycloak-operator
# MGDAPI-3902 implement flag for disabling syncing of the replicas

## Related GH Issue
Closes: https://github.com/keycloak/keycloak-operator/issues/549
Jira: https://issues.redhat.com/browse/MGDAPI-3902

## Additional Information
For RHOAM, we would like to introduce a flag in the kc CR to disable syncing replicas between keycloak CR and stateful sets.

## Verification Steps
- create new kc instance with the flag `kc.Spec.DisableReplicasSyncing` set to true.
- on creation of the instance verify that the replicas count is at 0
- update kc instances to 2 via kc cr
- verify that the stateful set remains at 0

## Checklist:
